### PR TITLE
Update use-kustomize-for-kubernetes-deployments.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
Added Command Flag Support

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md [hunk 1]
```diff
@@ -462,6 +462,32 @@ The Harness Delegate ships with the 3.5.4 [release](https://github.com/kubernete
 
 If you want to use a different release of Kustomize, add it to a location on the Delegate, update the following Delegate files, and restart the Delegate.
 
+
+## Configuring a Kustomize Service with Command Flags
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:466]:
```suggestion
## Configuring a Kustomize service with command flags 
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md [hunk 2]
```diff
@@ -462,6 +462,32 @@ The Harness Delegate ships with the 3.5.4 [release](https://github.com/kubernete
 
 If you want to use a different release of Kustomize, add it to a location on the Delegate, update the following Delegate files, and restart the Delegate.
 
+
+## Configuring a Kustomize Service with Command Flags 
+
+Command Flags lets users change the behavior of how Harness will perform a Kustomize based deployment. With the `build` command you can pass in sub commands to change the behavior of the `kustomize build` command.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:468]:
```suggestion
Command flags let users change the behavior of how Harness performs a Kustomize based deployment. Using the `build` command, you can pass sub-commands to change the behavior of the `kustomize build` command. 
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md [hunk 3]
```diff
@@ -462,6 +462,32 @@ The Harness Delegate ships with the 3.5.4 [release](https://github.com/kubernete
 
 If you want to use a different release of Kustomize, add it to a location on the Delegate, update the following Delegate files, and restart the Delegate.
 
+
+## Configuring a Kustomize Service with Command Flags 
+
+Command Flags lets users change the behavior of how Harness will perform a Kustomize based deployment. With the `build` command you can pass in sub commands to change the behavior of the `kustomize build` command. 
+
+Example Commands:
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:470]:
```suggestion
Here's a sample command:
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md [hunk 4]
```diff
@@ -462,6 +462,32 @@ The Harness Delegate ships with the 3.5.4 [release](https://github.com/kubernete
...
+```TEXT
+## To print the stack trace of the kustomize build command 
+kustomize build --stack-trace 
+```
+
+You can deploy [Helm Charts with Kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/chart.md#helm-related-flags), Harness supports this today with the proper configuration. Please see the example:
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:476]:
```suggestion
Harness also supports [Helm Charts with Kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/chart.md#helm-related-flags) deployments. 

For example:
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md [hunk 5]
```diff
@@ -462,6 +462,32 @@ The Harness Delegate ships with the 3.5.4 [release](https://github.com/kubernete
...
+
+
+```Text
+## Enable the Helm Template and Pull capabilities and render your kustomization with Helm. 
+kustomize build --enable-helm --helm-command `<YOUR_HELM_COMMAND>`
+```
+
+
+
+### Limitation
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:486]:
```suggestion
### Limitations
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md [hunk 6]
```diff
@@ -462,6 +462,32 @@ The Harness Delegate ships with the 3.5.4 [release](https://github.com/kubernete
...
+
+Harness only supports the Kustomize `build` command. The Kustomize `build` command performs the following: `build a kustomization target from a directory or URL.`
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:488]:
```suggestion
Harness supports the Kustomize `build` command only. The Kustomize `build` command performs the following: `build a kustomization target from a directory or URL.` 
```

Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:488]: @thisrohangupta Is this a command, `build a kustomization target from a directory or URL.` ? Or did you mean, the Kustomize `build` command builds a Kustomization target from a directory or URL?

Comment (thisrohangupta) [reply docs/continuous-delivery/deploy-srv-diff-platforms/kustomize/kustomize-howtos/use-kustomize-for-kubernetes-deployments.md:491]: the Kustomize build command builds a Kustomization target from a directory or URL - thats what i meant to say

**Review by roshnisarangadharan — APPROVED** [review]

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request closed without merging**